Tools/_sunrise/changelog/changelog_actions.py

```
@@ -306,12 +543,20 @@ def parse_pr_body(
         media_match = MEDIA_RE.match(line)
         if media_match:
-            category = entries.setdefault(current_category, {"changes": [], "media": []})
-            if not category["changes"]:
-                raise ValueError(f"категория {current_category} содержит медиа, но не содержит записей изменений")
-            media = parse_media_value(media_match.group(1))
-            media["change"] = len(category["changes"]) - 1
-            category["media"].append(media)
+            if allow_inline_media:
+                category = entries.setdefault(current_category, {"changes": [], "media": []})
+                if not category["changes"]:
+                    raise ValueError(f"категория {current_category} содержит медиа, но не содержит записей изменений")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)